rust/osm2lanes/src/transform/tags_to_lanes/bicycle.rs

```diff
@@ -0,0 +1,125 @@
+use super::*;
+
+pub fn bicycle(
+    tags: &Tags,
+    locale: &Locale,
+    oneway: bool,
+    forward_side: &mut Vec<Lane>,
+    backward_side: &mut Vec<Lane>,
+    warnings: &mut RoadWarnings,
+) -> ModeResult {
+    impl Tags {
```

Comment:
Rust is crazy -- I didn't realize you could add a method to a struct just in the middle of some other method! Any reason to not put this outside of `fn bicycle` and just in this module scope?

Reply:
When I implemented this, I assumed this falls under the general programming "smallest scope is better".

In this case though after having copied this function into its own file, I agree this impl should sit next to the function